### PR TITLE
Add shield of the pastalord and stuff

### PR DESCRIPTION
**Pull request by spaghetti-squash**

### CODE REVIEW

File: src/passenger.ts [hunk 1]
```diff
@@ -20,9 +20,13 @@ const passenger: CrimboQuest = {
         return chooseQuestOutfit({ location, isFree: false }, drunkSpec);
       },
       effects: () =>
-        $effects`Blood Bubble, Blood Bond, Frenzied\, Bloody, Empathy, Leash of Linguini, Ruthlessly Efficient, Mathematically Precise`.filter(
-          (effect) => have(toSkill(effect))
-        ),
+        [
+          ...$effects`Blood Bubble, Blood Bond, Frenzied\, Bloody, Empathy, Leash of Linguini, Ruthlessly Efficient, Mathematically Precise, Psalm of Pointiness`,
+          toEffect($skill`Shield of the Pastalord`),
```
Comment (docrostov) [inline src/passenger.ts:25]: i think this is good to have but my one small callout is that this is pretty MP regen heavy and we probably need a few things that restore MP in our outfit to use this. i think it is very worth adding tho.

File: src/macro.ts [hunk 2]
```diff
@@ -130,15 +129,14 @@ export default class Macro extends StrictMacro {
...
     return this.externalIf(
       myPrimestat() === $stat`mysticality`,
       Macro.tryHaveSkill($skill`Stuffed Mortar Shell`)
-        .trySkillRepeat(bestKillSpell)
-        .trySkillRepeat(otherKillSpell)
+        .externalIf(
+          myClass() === $class`Pastamancer` && have($skill`Fearful Fettucini`),
+          Macro.trySkillRepeat($skill`Fearful Fettucini`)
+        )
+        .trySkillRepeat($skill`Saucegeyser`)
```
Comment (docrostov) [inline src/macro.ts]: only do this if we're adding a lantern of some sort as well. also, fearful is only better if you have and are equipping the velour veil, so make that a condition here too

File: src/setup.ts [hunk 3]
```diff
@@ -112,6 +119,27 @@ export const setup: Quest<CrimboTask> = {
...
       sobriety: "sober",
     },
+    {
+      name: "MCD",
+      completed: () => !currentMcd(),
+      do: () => changeMcd(0),
+      sobriety: "either",
+    },
+    {
+      name: "The Captain",
```
Comment (docrostov) [inline src/setup.ts:129]: hell yeah this shit is good

File: src/macro.ts [hunk 4]
```diff
@@ -131,12 +131,11 @@ export default class Macro extends StrictMacro {
 
     return this.externalIf(
       myPrimestat() === $stat`mysticality`,
-      Macro.tryHaveSkill($skill`Stuffed Mortar Shell`)
-        .externalIf(
-          myClass() === $class`Pastamancer` && have($skill`Fearful Fettucini`),
-          Macro.trySkillRepeat($skill`Fearful Fettucini`)
-        )
-        .trySkillRepeat($skill`Saucegeyser`)
+      Macro.tryHaveSkill($skill`Stuffed Mortar Shell`).externalIf(
```
Comment (docrostov) [inline src/macro.ts]: i would take out the shell i think? not sure. one of the nice things about spellslinging is that half your combats you don't get hit at all. forcing one hit a fight may be part of why railo is having combat trouble

File: src/macro.ts (hunk 2 above)
Comment (spaghetti-squash) [reply src/macro.ts]: I think it is unlikely we will be adding velour veil or lanterns, but who could know

**Comment by docrostov** [comment]
seems fine. i will state for posterity that i think it is a little silly that we are refusing to oust crown. one instance of beaten up is like 20 turns of random marginal equipment in the hat slot. bjorn is "good" but this is not a profitable script, until we do libram stuff with the new accessory. then maybe it will be profitable.

**Pull request merged**